website/docs/concepts/nodes-networks.md

```diff
@@ -91,12 +91,12 @@ Every network's execution layer works with (and only with) its corresponding "pa
     <tr>
       <td>Goerli</td>
       <td>Prater</td>
-      <td>The Goerli-Prater pair is the test network that most people use when learning how to configure their validator for the first time. After Sepolia, Goerli-Prater will be Merge-tested.<br/><br/>This network pair mints and manages <strong>Goerli ETH</strong>, a type of testnet ETH used exclusively within this network pair.</td>
+      <td>The Goerli is the test network that most people use when learning how to configure their validator for the first time.<br/><br/>This network mints and manages <strong>Goerli ETH</strong>, a type of testnet ETH used exclusively within this network.</td>
     </tr>
     <tr>
       <td>Sepolia</td>
       <td>Sepolia</td>
-      <td>Consensus-layer Sepolia is a new network that was created to facilitate testing. The <a href='../install/install-with-script'>Prysm Quickstart</a> shows you how to configure a Merge-ready node on Sepolia. Note that this is a permissioned network, so you can run a node on Sepolia, but not a validator.<br/><br/>This network pair mints and manages <strong>SepplETH</strong>, a type of testnet ETH used exclusively within this network pair.</td>
+      <td>Sepolia is a network that was created to facilitate testing. The <a href='../install/install-with-script'>Prysm Quickstart</a> shows you how to configure a node on Sepolia. Note that this is a permissioned network, so you can run a node on Sepolia, but not a validator.<br/><br/>This network pair mints and manages <strong>SepplETH</strong>, a type of testnet ETH used exclusively within this network pair.</td>
```

Comment:
facilitate smart contract testing where as holesky and goerli are for protocol and staking testing (speaking of which we should add a todo for holesky updates on the docs)